lgc/CMakeLists.txt

```diff
@@ -163,6 +163,7 @@ target_include_directories(LLVMlgc
 PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/interface>
     $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/interface>
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>
```

Comment:
This is where the patch gets complicated. Instead of such a brittle include path, it should rather be a dependency on vkgc_headers. However, I think the currently missing dependency between vkgc and lgc is purposefully.

@trenouf, @nhaehnle any ideas? Do we need another cmake project for 20 lines of code?

Reply:
It’s for use in the vkgc header, see the discussion here: https://github.com/GPUOpen-Drivers/llpc/pull/2645#discussion_r1309842484

Reply:
Ah, ok, thanks. Sounds like it should go here then (or just in the vkgc.h file rather than in its own file), but the existing def and use in LGC should remain separate, instead of trying to share this one.